contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,1020 @@
+        </tr>
+        <tr>
+            <td><strong>In-app prompts and messages</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send messages to users in your app.</td>
+        </tr>
+        <tr>
+            <td><strong>Surveys</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <tr>
+            <td><strong>Customer interview scheduling</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>In-app widget for requesting customer interviews.</td>
+        </tr>
+        <tr>
+            <td><strong>User interview recordings and transcriptions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Apps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Push and pull data to other destinations</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Session replays:** [Session replays](/session-replay) in PostHog recreate exactly what real users see and how they use your product. They also enable you to debug problems using built-in [console logs](/docs/session-replay/configure#console-logs-recording-beta), [performance monitoring](/tutorials/performance-metrics) and the [DOM explorer](/blog/posthog-changelog#experimental-dom-explorer-mode).
+
+- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
+
+- **A/B testing:** In PostHog, you can use the [experimentation suite](/ab-testing) to create multivariate tests within your product, such as showing some users a different page layout to others. Over time, you can build an understanding of which page performs better, correlate results with other events, and deploy a final version.
+
+
+### Session replays
+
+Session replays are an essential tool for understanding how people use your product, especially for [early-stage companies](/blog/early-stage-analytics) searching for product-market fit. 
+
+> Learn [how Contra used session replays](/customers/contra) to increase registrations by 30%. 
```

Comment:
```suggestion
> **Ready to learn more?** Find out [how Contra used PostHog's session replays](/customers/contra) to increase registrations by 30%. 
```
Was expecting us to work it into the copy, but I like this CTA. Will defer to @andyvan-ph on if it's too heavy-handed though, in which case we can just lose the `>`